osgtest/tests/test_84_xrootd.py

```diff
@@ -12,6 +12,5 @@ def test_01_stop_xrootd(self):
             files.restore('/etc/grid-security/xrd/xrdmapfile', "xrootd")
         core.skip_ok_unless_installed('xrootd', by_dependency=True)
         self.skip_ok_if(core.state['xrootd.started-server'] == False, 'did not start server')
-
-        service.check_stop(core.config['xrootd_service'])
+        service.stop(core.config['xrootd_service'])
```

Comment:
It was taking longer than 30 seconds in my previous attempts, I didn't try anything past that.